library/block/enchantment_stuff/src/main/java/org/quiltmc/qsl/block/enchantmentstuff/impl/BlockEnchantmentStuffInitializer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.block.enchantmentstuff.impl;
+
+import net.minecraft.block.Blocks;
+import org.quiltmc.loader.api.ModContainer;
+import org.quiltmc.qsl.base.api.entrypoint.ModInitializer;
+import org.quiltmc.qsl.block.enchantmentstuff.api.EnchantmentPowerAttachment;
+
+public class BlockEnchantmentStuffInitializer implements ModInitializer {
+	@Override
+	public void onInitialize(ModContainer mod) {
+		EnchantmentPowerAttachment.POWER_LEVEL.put(Blocks.BOOKSHELF, 1f);
```

Comment:
It might be better to put this in a static initializer in `EnchantmentPowerAttachment`, so mods can properly override the built-in value without having to worry about load order shenanigans.

Maybe I should add a `with(K, V)` method to the attachment builder...

Reply:
I thought you weren't supposed to access the `Blocks` class before the initializer?

Reply:
Hm, seems like `block_content_registry` also waits for the initializer to copy the vanilla entries into the attachment. This is probably fine for now then, I'll look into engineering a solution later.